Automattic/jetpack
# Masterbar: Use Calypso import/export only on simple sites

Props @JoshuaGoode.
Fixes #18294.

#### Changes proposed in this Pull Request:

* Moves Import/Export removal & addition to `class-wpcom-admin-menu.php`.
* Removes unneeded unit tests for `class-admin-menu.php`.
* Updates unit tests for `class-atomic-admin-menu.php` to make sure we get the expected results.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Install the Jetpack Beta plugin on your WoA site.
* Switch Jetpack Beta to using this branch
* Make sure Tools > Import and Tools > Export point to their wp-admin pages.

#### Proposed changelog entry for your changes:


* None needed.
